### PR TITLE
Update contributing local testing section + arrange scripts alphabetically

### PR DESCRIPTION
**Pull request by ranyitz**
### 🔦 Summary
Arrange `package.json` scripts and update contributing local testing section.

* Add a command to run all monorepo tests at once. (not including `e2e`)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,7 +17,10 @@ Please **ask first** if somebody else is already working on this or the core dev
 
 That's it, you're good to go.
 
-* `npm test` - Run yoshi's tests.
+* `npm test-all` - Run all monorepo tests.
+* `npm test-yoshi` - Run `yoshi` package tests.
+* `npm test-create-yoshi-app` - Run `create-yoshi-app` package tests.
+* `npm test-configs` - Run all packages tests besides `yoshi` & `create-yoshi-app`.
 * `npm run E2E` - create all yoshi projects and install, build and test each one of them.
```
Comment (eddierl) [inline CONTRIBUTING.md]: on the way change it to `e2e`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -17,7 +17,10 @@ Please **ask first** if somebody else is already working on this or the core dev
 
 That's it, you're good to go.
 
-* `npm test` - Run yoshi's tests.
+* `npm test-all` - Run all monorepo tests.
```
Comment (eddierl) [inline CONTRIBUTING.md]: change `npm` to `npm run`

File: CONTRIBUTING.md (hunk 1 above)
Comment (ranyitz) [reply CONTRIBUTING.md]: Nice catch!

File: CONTRIBUTING.md (hunk 2 above)
Comment (ranyitz) [reply CONTRIBUTING.md]: Thanks!

**Pull request merged**